.dependabot/config.yml

```diff
@@ -0,0 +1,85 @@
+version: 1
+update_configs:
+
+  # dotnet
+  - package_manager: "dotnet:nuget"
+    directory: "/builders/testdata/dotnet/functions/cs_multiple_targets/"
+    update_schedule: "live"
+  - package_manager: "dotnet:nuget"
+    directory: "/builders/testdata/dotnet/functions/cs_single_target/"
+    update_schedule: "live"
+  - package_manager: "dotnet:nuget"
+    directory: "/builders/testdata/dotnet/functions/fs_function/"
+    update_schedule: "live"
+  - package_manager: "dotnet:nuget"
+    directory: "/builders/testdata/dotnet/functions/vb_function/"
+    update_schedule: "live"
+
+  # go
+  - package_manager: "go:modules"
+    directory: "/builders/testdata/go/functions/with_framework"
+    update_schedule: "live"
+  - package_manager: "go:modules"
+    directory: "/builders/testdata/go/functions/with_subdir"
+    update_schedule: "live"
+
+  # java
+  #- package_manager: "java"
+  #  directory: "/"
+  #  update_schedule: "live"
+
+  # nodejs
+  - package_manager: "javascript"
+    directory: "/builders/testdata/nodejs/functions/load_user_dependencies"
```

Comment:
I'm worried the testdata directories will go out of date over time and the automated PRs will lead to a false sense of security that all tests have been updated and potentially leaving some of them outdated. Thoughts on this?

Does this update lockfiles as well? I notice that even though the package.json specify the latest version of the framework, we have lockfiles that haven't been updated for a few verisons.

Reply:
> I'm worried the testdata directories will go out of date over time and the automated PRs will lead to a false sense of security that all tests have been updated and potentially leaving some of them outdated. Thoughts on this?

I agree that it's possible that this might not capture new tests. Unfortunately the `directory` configuration does not support wildcards (https://github.com/dependabot/feedback/issues/408) so there isn't a better way to address this.

However I think this is still better than the alternatives: that Function Frameworks maintainers would need to remember to manually bump the versions in this repo, or the maintainers of this repo would need to know when each individual framework releases a new version, or that all these tests just slowly become more and more out of date.

> Does this update lockfiles as well? I notice that even though the package.json specify the latest version of the framework, we have lockfiles that haven't been updated for a few versions.

I can't speak to the behavior in all languages, but I would expect it to update the lockfiles as well, as long as the requirements files are not over-constrained.

Reply:
The alternative would be to only update the ff buildpack to create "notification" PRs and then have a human update everything else, but that also seems brittle. Maybe  we could add a note for reviewers to check if all tests have been updated using https://dependabot.com/docs/config-file/#commit_message?

In either case, this is great, thanks for the PR!